modules/console/pages/config/security/authorization.adoc

```diff
@@ -1,21 +1,130 @@
 :page-aliases: console:single-sign-on/authorization.adoc, manage:security/console/authorization.adoc, console:features/role-bindings.adoc, console:reference/role-bindings.adoc, reference:console/role-bindings.adoc
 
-Redpanda Console uses role-based access control (RBAC) to manage and restrict access to features in the user interface. RBAC allows you to control access to different resources in Redpanda Console by binding roles (admin, editor, viewer) to users. These role bindings determine the permissions that users have when interacting with Redpanda Console.
-
 [NOTE]
 ====
 include::shared:partial$enterprise-and-console.adoc[]
 ====
 
-RBAC behavior depends on your authentication configuration in Redpanda Console. When using static credentials, you must give users permissions to use Redpanda Console by defining RBAC rules in the Redpanda Console configuration file. If authentication for an API is configured to impersonate users, Redpanda Console RBAC rules are ignored for that API and permissions are derived only from the roles assigned in Redpanda. For more information, see the xref:console:config/security/authentication.adoc[].
+{description}
+
+Redpanda Console supports two authorization modes, depending on whether *user impersonation* is enabled in the authentication configuration:
+
+[cols="1,2"]
+|===
+| Mode | Who Evaluates Permissions?
+
+| Impersonation disabled (`impersonateUser: false`)
+| Redpanda Console evaluates permissions using role bindings defined in its configuration. Redpanda sees all requests as coming from a static service account.
+
+| Impersonation enabled (`impersonateUser: true`)
+| Redpanda evaluates permissions using its internal RBAC and ACLs. The role bindings in the Redpanda Console configuration are ignored.
+|===
+
+For more information about authentication options, see the xref:console:config/security/authentication.adoc[Redpanda Console authentication].
+
+== Modes of service account usage
+
+*When impersonation is disabled*, Redpanda Console uses a static service account to connect to Redpanda APIs. This service account must be listed as a superuser principal in the Redpanda `superusers` configuration.
+
+In Redpanda, a principal is the authenticated identity (such as a username or OIDC subject) used for access control. For details, see xref:manage:security/authentication.adoc#principals[Users, Principals, and Superusers].
+
+IMPORTANT: For any secret values, xref:console:config/configure-console.adoc[use environment variables] instead of hardcoding them in the configuration file. For example, use `KAFKA_SASL_PASSWORD` for the service account password.
+
+There are two ways to use this service account:
+
+=== Static service account only (no login)
+
+Use this mode to connect Redpanda Console to Redpanda with a static account and *no user login*.
+
+- No login screen.
+- All users share the same access level.
+- Redpanda Console roles and RBAC are not enforced.
+- Useful for internal tools.
```

Comment:
A bit confused what this list is describing. Seems like it needs an intro to explain what is being listed.